docs/source/usage/deployment.rst

```diff
@@ -188,7 +196,7 @@ This is where we take everything we've done up to this point and start the deplo
      7. For stack parameters, enter the values decided in :ref:`Getting Started` Step 2.
        * For service, enter your <service> name that you determined earlier.
        * For environment, choose “dev”, "staging", or "prod" according to the environment you are working out of.
-       * In the TOKEN_ARN field, you'll need to enter your token arn to correctly map the IAM permissions for this resource.
+       * In the SSMParamName field, you'll need to enter the name of the SSM parameter used to store the auth token, to correctly map the IAM permissions for this resource.
```

Comment:
Same issue as referenced above in regards to the commas.
Just a grammar nitpick.

Reply:
Done.